q2_feature_table/plugin_setup.py

```diff
@@ -652,3 +654,62 @@
                 'splits are defined by values in metadata column.',
     examples={}
 )
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_feature_frequencies,
+    inputs={'table': FeatureTable[Frequency | PresenceAbsence |
+                                  RelativeFrequency]},
+    parameters={},
+    outputs={'feature_frequencies': ImmutableMetadata},
+    input_descriptions={
+        'table': 'The input feature table.'
+    },
+    output_descriptions={
+        'feature_frequencies': 'Per-sample and total frequencies per feature.'
+    },
+    name='Tabulate feature frequencies',
+    description='Tabulate feature count and total frequency per sample.',
+    examples={'feature_table_tabulate_feature_frequencies':
+              ex.feature_table_tabulate_feature_freqs}
+)
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_sample_frequencies,
+    inputs={'table': FeatureTable[Frequency | PresenceAbsence |
+                                  RelativeFrequency]},
+    parameters={},
+    outputs={'sample_frequencies': ImmutableMetadata},
+    input_descriptions={
+        'table': 'The input feature table.'
+    },
+    output_descriptions={
+        'sample_frequencies': 'Per-feature and total frequencies per sample.'
```

Comment:
```suggestion
        'sample_frequencies': 'Observed feature count and total frequencies per sample.'
```

You can use this as the description for output from `summarize-plus` as well.